### PR TITLE
Expose instance startup time in worker info

### PR DESCRIPTION
**Pull request by meatballhat**

### CODE REVIEW

**Comment by henrikhodne** [comment]
Looks good! :+1: :sparkles:

File: backend/fake.go [hunk 1]
```diff
@@ -23,13 +24,15 @@ func newFakeProvider(cfg *config.ProviderConfig) (Provider, error) {
 }
 
 func (p *fakeProvider) Start(ctx context.Context, _ *StartAttributes) (Instance, error) {
-	return &fakeInstance{p: p}, nil
+	return &fakeInstance{p: p, startupDuration: 42170 * time.Millisecond}, nil
```
Comment (moul) [inline backend/fake.go]: Hey @meatballhat, is `42170` a test value ?

Comment (meatballhat) [inline backend/fake.go]: @moul Confirm, it is.  Injecting it in the tests is probs better.  I'll fix. :heart_eyes_cat:

File: step_open_log_writer.go [hunk 2]
```diff
@@ -49,8 +49,9 @@ func (s *stepOpenLogWriter) writeUsingWorker(state multistep.StateBag, w io.Writ
 		_, _ = writeFold(w, "worker_info", []byte(strings.Join([]string{
 			"\033[33;1mWorker information\033[0m",
 			fmt.Sprintf("hostname: %s", hostname),
-			fmt.Sprintf("version: %s", VersionString),
+			fmt.Sprintf("version: %s https://github.com/travis-ci/worker/tree/%s", VersionString, RevisionString),
```
Comment (meatballhat) [inline step_open_log_writer.go]: @henrikhodne Any qualms with this?

Comment (henrikhodne) [inline step_open_log_writer.go]: Not really. It felt a little strange to put the URL in the code like that, but I think it's the right way to do it.

Comment (meatballhat) [inline step_open_log_writer.go]: ooh, ooh, I have an idear

**Pull request merged**